### ISSUE TITLE
git rebase using vim as an editor crashes conemu

### ISSUE THREAD

**Issue by ConEmu** [ConEmu/old-issues#1059]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on May 07, 2013 10:12:17_

OS version: Win7 x64 SP1 ConEmu version: ConEmu .Maximus5.130427

Far version (if you are using Far Manager): Not using FAR

Steps to reproduction
1. Install vim 7.3, add to PATH, set as "core.editor" in git (git config --global core.editor vim)
2. In a git repository, interactively rebase against master:

git rebase -i master
1. Crash usually occurs.

I'll try to build conemu from source and debug this myself.

_Original issue: http://code.google.com/p/conemu-maximus5/issues/detail?id=1060_

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on May 07, 2013 10:19:47_

Well, lets try. Use ConEmu -preview brunch. It is latest ATM.
Another way - create memory dumps of all spawned processes at the moment off the crash.
It may be easily created with
ConemuC /debugtree ConEmu .exe ...

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on May 07, 2013 10:21:34_

Thanks for the details! I'll do that as soon as I have time.

**Comment by ConEmu** [comment]
_From [mark.mac...@gmail.com](https://code.google.com/u/109500556076519976929/) on May 08, 2013 16:21:16_

I also use VIM 7.3 that comes with msysgit. ConEmu 120727c x64 crashes for me intermittently just running VIM 7.3 repeatedly. This occues on the order of one in 100 attempts, when I run vim.exe in a loop (e.g., "for /L %i in (1 1 100) do echo %i && vim.exe") it crashed for me on the 73rd run. When I'm not counting it feels like it can happen as often as 1 in 10.

An unfortunate side-effect when this happens is that it leaves vim.exe still running, and that must be killed in task manager.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on May 08, 2013 16:25:52_

It's incorrect to compare behavior of stable and current alpha/preview versions. Very large changes count was made. If this reproduces in last preview - let me know.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on May 08, 2013 16:54:08_

I haven't had luck yet building conemu from source. I have VS 2010 and 2012 installed but am not too familiar with building C++ apps from source. Would you recommend I use mingw?

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on May 08, 2013 16:58:24_

FYI I'd like to be able to F5 and start conemu from VS in debug mode. I have the " ConEmu " project set as the start project, but when I start in Debug mode I get an error. Screen shot is attached.

**Attachment:** [conemu-start-error.png](http://code.google.com/p/conemu-maximus5/issues/detail?id=1060#hc6)

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on May 08, 2013 16:58:54_

for my taste, mingw has poor debug features... you may try, of course, but not recommended.

What is the problem? Open CE10.sln and press build. Debug_C configuration for example.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on May 08, 2013 16:59:48_

OK trying that now. Thanks!

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on May 08, 2013 17:00:20_

You are on Win64, so you must build x64 target too

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on May 08, 2013 17:01:28_

You may use 2012 too, if you want. Doesn't matter.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on May 08, 2013 17:02:50_

One build error:

Error   3   error LNK2001: unresolved external symbol GetGlobalInfoW    C:\proj\src\conemu-maximus5-read-only\ ConEmu -preview\src\ConEmuBg\export.def  1   1   Far.ConEmuBg

Debug_C configuration, x64 platform.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on May 08, 2013 17:05:23_

You have no need in ConEmuBg

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on May 08, 2013 17:08:46_

Well, if I right-click on " ConEmu " and choose Debug -> Start New Instance, I get this in the build output:

1>------ Build started: Project: ConEmu , Configuration: Debug_C x64 ------
1>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\Microsoft.CppBuild.targets(990,5): warning MSB8012: TargetPath(C:\Temp\CE\ ConEmu .x64.Debug_C\ ConEmu .exe) does not match the Linker's OutputFile property value (C:\ ConEmu \ConEmu64.exe). This may cause your project to build incorrectly. To correct this, please make sure that $(OutDir), $(TargetName) and $(TargetExt) property values match the value specified in %(Link.OutputFile).
1>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\Microsoft.CppBuild.targets(992,5): warning MSB8012: TargetName( ConEmu ) does not match the Linker's OutputFile property value (ConEmu64). This may cause your project to build incorrectly. To correct this, please make sure that $(OutDir), $(TargetName) and $(TargetExt) property values match the value specified in %(Link.OutputFile).
1>  ConEmu10.vcxproj -> C:\Temp\CE\ ConEmu .x64.Debug_C\ ConEmu .exe

And the attached error message.

**Attachment:** [conemu-start-error-2.png](http://code.google.com/p/conemu-maximus5/issues/detail?id=1060#hc14)

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on May 08, 2013 17:13:52_

Ignore warnings.
Change debug path in ConEmu project properties
It will be c:\ ConEmu \ ConEmu .exe

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on May 08, 2013 17:19:33_

Getting farther. When I click "Retry" cmd.exe crashes. If I keep clicking "Cancel" I eventually get to the running command prompt.

I get the same popup when changing directories, or running a git command. Basically, when output goes to the screen.

**Attachment:** [conemu-start-error-3.png conemu-assert-git-1.png](http://code.google.com/p/conemu-maximus5/issues/detail?id=1060#hc16)

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on May 08, 2013 17:23:03_

thought, it will be easier to use /debugtree mentioned before. With official preview build off course.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on May 08, 2013 17:24:06_

OK I will do that. I can't seem to get the crash to happen this way anyway.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on May 09, 2013 08:49:13_

Well, /debugtree didn't do much but I did get dumps from task manager of all the running ConEmu processes at the time of a crash, as well as some hopefully useful output from WinDbg.

Next I'm going to try the instructions here to run ConEmu from within WinDbg to get some info for you: https://developer.mozilla.org/en-US/docs/How_to_get_a_stacktrace_with_WinDbg I'm assuming I can set my symbol path to my local ConEmu build, unless you have symbols you can provide for the 130427 build.

Dump files here: https://www.dropbox.com/s/df1slnvzz1i7yqg/ConEmu-Dumps.7z

**Attachment:** [conemu-debug_2390_2013-05-09_08-35-43-079.log](http://code.google.com/p/conemu-maximus5/issues/detail?id=1060#hc19)

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 12, 2013 07:26:03_

More information:

I've switched from Windows 7 to Windows 8 for my day-to-day development use. I upgraded to ConEmu 130604 and interactive rebase in git using vim.exe does NOT appear to crash anymore.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 12, 2013 07:30:26_

Well that was short-lived. In the same window I did the git rebase, I later executed vim.exe by itself and the window crashed.

How can I help debug this? I can't seem to build ConEmu reliably on my machine. If you supply a build with debug symbols, I can run ConEmu within WinDBG to get you more information.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 12, 2013 07:34:19_

Actually, every ConEmu build is created with debug information!
Just create full memory dump of crashed process and upload it somewhere (googlecode doesn't accept large files)

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 12, 2013 07:34:53_

Issue 1061 has been merged into this issue.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 12, 2013 07:53:49_

OK, I'll get a new memory dump.

Were you able to get information from the dump I provided on May 9th above? https://www.dropbox.com/s/df1slnvzz1i7yqg/ConEmu-Dumps.7z

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 12, 2013 08:13:48_

@luke,
You've created dumps for ConEmu64.exe and ConEmuC64.exe processes.
But you are talking about "crash". Where is this crash?
From your description, seems like crash occures in vim.exe process.
There is no dump for this process in your archive.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 12, 2013 09:51:38_

vim.exe doesn't crash - I have to manually kill the vim.exe process when the conemu window crashes.

Next time it happens I'll see what I can get from the "conemu has stopped working" popup window.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 12, 2013 09:55:30_

Actually, the crash may occurs in the buggy conhost.exe...
Does your problem repeated constantly? Or from time to time?
Next time crash occurs, please make screenshot and take a look at conhost.exe presence. And dumps of course...

**Comment by ConEmu** [comment]
_From [boxmon...@gmail.com](https://code.google.com/u/102446199106730395162/) on June 12, 2013 10:52:13_

This happens to me as well. http://cl.ly/image/0E0b0V3R0c1q Problem signature:
  Problem Event Name:   APPCRASH
  Application Name: ConEmuC64.exe
  Application Version:  13.6.4.0
  Application Timestamp:    51ae785c
  Fault Module Name:    ntdll.dll
  Fault Module Version: 6.1.7601.17725
  Fault Module Timestamp:   4ec4aa8e
  Exception Code:   c0000005
  Exception Offset: 000000000004e99a
  OS Version:   6.1.7601.2.1.0.256.48
  Locale ID:    1033
  Additional Information 1: 45b6
  Additional Information 2: 45b6fee40660715107b63136962bb724
  Additional Information 3: 4c8a
  Additional Information 4: 4c8a8978165a554136df9283a549c878

...also when I do:
ConemuC /debugtree Conemu.exe
I get this error:
Parsing command line failed (/C argument not found):
"c:\Program Files\ ConEmu \ ConEmu \ConemuC.exe" "C:/Program Files (x86)/Git/debugtree" Conemu.exe

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 12, 2013 12:34:20_

My crash is not constant. I get the same crash information as above - ConEmuC64.exe.

I'm uploading dump files to dropbox now.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 12, 2013 13:13:38_

Dump files - there were two ConEmuC64 processes at the time of the crash so I dumped them both: https://www.dropbox.com/s/9praykt0q8pnyyh/ConEmuC64-1.7z https://www.dropbox.com/s/aag1l7ug1s1ei1f/ConEmuC64-2.7z

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 12, 2013 13:18:54_

And what about conhost?

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 12, 2013 13:48:10_

There was no indication of any kind that conhost.exe had crashed or had an issue. After I received the popup showing similar error info as comment `#28`, I created the dumps. Then I had to use Task Manager to kill vim.exe

**Comment by ConEmu** [comment]
_From [boxmon...@gmail.com](https://code.google.com/u/102446199106730395162/) on June 12, 2013 14:02:05_

This also leaves an orphaned sh.exe running as well obviously but I don't know of a good way to figure out which one needs to be killed vs which ones are still attached to conemu.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 12, 2013 14:03:27_

I see no exception in both attached dumps. They works as expected.

Seems like crashed process was not dumped by you. I don't know the reason.
At least, show screenshot of Info page in ConEmu while crash occurs.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 12, 2013 14:17:45_

My screen looks like like comment `#28` -

Problem signature:
  Problem Event Name:   APPCRASH
  Application Name: ConEmuC64.exe
  Application Version:  13.6.4.0
  Application Timestamp:    51ae785c
  Fault Module Name:    ntdll.dll
  Fault Module Version: 6.1.7601.17725

Should I provide dumps of all ConEmu *.exe, conhost.exe and vim.exe processes when this issue happens? I can get it to happen outside of a "git rebase" so no git.exe or sh.exe processes are involved.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 12, 2013 14:19:34_

Yes, all dumps 

AND

"show screenshot of Info page in ConEmu while crash occurs."
"Info" is the page of "Settings" dialog

**Comment by ConEmu** [comment]
_From [kevin.ca...@bromium.com](https://code.google.com/u/101475368719670331332/) on June 12, 2013 14:47:09_

For what its worth, I've been unable to repro lately after I set the environment variable TERM=msys.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 12, 2013 18:40:38_

#37 - Kevin, quick question. Are you using vim from the official distribution? I'll try that setting out - thanks!

**Comment by ConEmu** [comment]
_From [kevin.ca...@bromium.com](https://code.google.com/u/101475368719670331332/) on June 13, 2013 02:37:11_

I'm not sure whether it launches the Vim from program files or another from
some git-specific path...

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 13, 2013 09:28:56_

OK I have the requested screenshots and will be uploading a file containing dumps of all the processes listed in the "Settings -> Info" dialog.

**Attachment:** [crash-popup.png settings-info.png](http://code.google.com/p/conemu-maximus5/issues/detail?id=1060#hc40)

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 13, 2013 09:34:16_

There is newer build ATM!

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 13, 2013 09:37:44_

#42 that's great had I known about it! I'll try it out.

In the meantime, here are the dumps: https://www.dropbox.com/s/hxchj0nrdk3cqra/Dumps.7z

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 13, 2013 09:46:36_
1. Enable autoupdate feature and you'll know about nee build almost immediately.
2. I'm not interested in old build dumps. New ConEmuC64 will (must) create crash dumps itself automatically. And it will inform user about it.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 13, 2013 09:48:12_

OK I've enabled auto update notifications.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 13, 2013 11:07:29_

OK I got a crash and a dump: https://www.dropbox.com/s/70bmz89dvonujd6/Trap-130612-2528.7z Also, conemu seems to leave conhost.exe processes behind even if a tab is closed. Is this expected?

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 13, 2013 11:10:02_

Conhost closing is not a ConEmu responsibility.
Moreover, ConEmu can't kill conhost :-)
Our will close automatically when all processes in console are closed.
If not - that may be a bug in conhost

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 13, 2013 11:11:22_

OK thanks for the explanation. I'm not familiar with the relationship between conhost.exe, conemu and other sub-processes.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 13, 2013 12:45:15_

On the first look it's an idiotic bug in kernel...
How can I reproduce this?

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 13, 2013 13:01:24_

Also, I'm interested in how often you can reproduce that?

Seems to me, required conditions are "192x50" console size
and slightly scrolled down (first visible line is 38, zero based).

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 13, 2013 13:20:36_

I can reproduce this every time I fire up conemu and then interact with git and vim in a powershell window. However I haven't come up with exact steps to reproduce just yet.

I have vim installed and C:\Program Files (x86)\vim\vim73 in my PATH. ( http://www.vim.org/download.php#pc ) 

I have git for windows installed and C:\Program Files (x86)\Git\bin in my PATH. I have vim set as the editor for git. ( https://code.google.com/p/msysgit/downloads/list?q=full+installer+official+git )

I have a conemu window set to maximized and running powershell. I then switch to a clone of a git repository and do at least one command that requires vim to run (like git rebase -i). After that if I run vim by itself to edit a file, I can get the crash. Previous builds crashed during the "git rebase" operation so this is a bit different.

Can I start ConEmu .exe via WinDBG and get you more data that way?

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 13, 2013 13:26:38_

WinDbg - no.

But you can do that:
Before starting vim - press CtrlWinAltSpace to show real console window
And reproduce the bug.
Ideally, I want to see video (screencast?) where both ConEmu and real console Windows are visible.
At least - screenshots before and after crash.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 13, 2013 13:54:38_

OK I'll try that out when I get a chance.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 14, 2013 01:16:48_

I think, with this small update nothing will changes. But try it. And attach dump, if crash occures.

**Attachment:** [ConEmuCD.130613a.7z](http://code.google.com/p/conemu-maximus5/issues/detail?id=1060#hc54)

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 14, 2013 05:23:21_

Got it! I'll see if I can get you screen shots.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 14, 2013 05:40:05_

Screen shots attached. It turns out that git starts up the msys vim so I'm going to switch that to use the vim in C:\Program Files and see if that changes anything. https://www.dropbox.com/s/wk409mtww0e2mlc/Trap-130613a-2872.7z

**Attachment:** [crash-popup.png full-screen.png real-console-window.png using-git-vim.PNG](http://code.google.com/p/conemu-maximus5/issues/detail?id=1060#hc56)

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 14, 2013 06:10:27_

OK I have something that consistently reproduces an exception in vim with that same exception (0xC0000005).

Start up ConEmu and open cmd.exe. Run vim and you'll see the welcome screen. Type the following keys:

:
%
s
/
CTRL-V
CTRL-M

Exception will happen. The above attempts to insert a literal CTRL-M character and for whatever reason that triggers it.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 14, 2013 06:16:04_

Can't reproduce exception in vim. Attach your vimrc?

And create dump of this exception

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 14, 2013 06:22:19_

Yep, it's something in my vim configuration. If I move the attached file and folder out of my user directory, those repro steps don't work. I'll start narrowing down the offending lines in the config.

**Attachment:** [vimstuff.7z](http://code.google.com/p/conemu-maximus5/issues/detail?id=1060#hc59)

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 14, 2013 06:34:21_

Another crash, not related to the above. https://www.dropbox.com/s/39rah2wgc5j4fj6/Trap-130613a-3844.7z

**Attachment:** [full-screen-2.PNG](http://code.google.com/p/conemu-maximus5/issues/detail?id=1060#hc60)

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 14, 2013 11:49:23_

Turn on "Inject ConEmuHk "

**Comment by ConEmu** [comment]
_From [boxmon...@gmail.com](https://code.google.com/u/102446199106730395162/) on June 14, 2013 12:35:46_

Not sure if this is still helpful but got another crash (they are very intermittent for me)

settings info: http://cl.ly/image/132C2U3k3Q05 exception:

Exception 0xC0000005 was occurred (ConEmu64 130612)

Memory dump was saved to
C:\Users\Jay\Desktop\ConEmuTrap\Trap-130612-9204.dmp

Please Zip it and send to developer https://code.google.com/p/conemu-maximus5/issues/entry dump file: https://dl.dropboxusercontent.com/u/3038213/Trap-130612-9204.zip

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 14, 2013 13:27:34_

@boxmonkey, what are you doing to get crash?

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 14, 2013 13:35:11_

@boxmonkey, you too, try to enable "Inject ConEmuHk "

**Comment by ConEmu** [comment]
_From [boxmon...@gmail.com](https://code.google.com/u/102446199106730395162/) on June 14, 2013 13:43:58_

That time I was just opening a file in vim with merge conflicts.

**Comment by ConEmu** [comment]
_From [boxmon...@gmail.com](https://code.google.com/u/102446199106730395162/) on June 14, 2013 13:45:01_

Will do! Do I need to close/reopen all tabs for it to take effect?

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 14, 2013 13:54:39_

Reopen tabs is preferred. Otherwise in some cases changing may not have effect.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 15, 2013 07:31:51_

Well I've been using ConEmu for about an hour this morning with "Inject ConEmuHk " enabled, and no crashes. I'll keep you updated. I'm assuming this setting won't be required once whatever bug is causing the crash is fixed.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 15, 2013 07:34:41_

The bug actually in microsoft kernel32.dll. How can I fix this bug? ;)

The only chance is to find some workaround, but I'm not sure it may even exists.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 25, 2013 06:40:17_

FYI I plan on using ConEmu full-time today. I'll let you know if I find this issue. I just got the latest alpha as well. Thanks for all the recent builds.

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 27, 2013 09:42:35_

Enabling " ConEmuHk " + the latest build appears to have resolved this issue. Thanks!

**Comment by ConEmu** [comment]
_From [boxmon...@gmail.com](https://code.google.com/u/102446199106730395162/) on June 27, 2013 09:48:25_

Same here, no crashes since!

**Comment by ConEmu** [comment]
_From [vladimir...@gmail.com](https://code.google.com/u/113088530674303484242/) on October 01, 2013 22:36:50_

I've downloaded version 130929. How to enable " ConEmuHk "? I cannot find this option.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on October 01, 2013 23:37:56_

Features page

**Comment by ConEmu** [comment]
_From [vladimir...@gmail.com](https://code.google.com/u/113088530674303484242/) on October 02, 2013 03:22:36_

Thank you! After update this issue is fixed for me.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on October 02, 2013 03:36:00_

**Status:** Fixed

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on March 12, 2014 12:50:26_

Issue 1508 has been merged into this issue.
